src/cf-core/src/cf-base.less

```diff
@@ -539,5 +539,38 @@ figure {
 }
 
 //
+// Code blocks
+//
+
+pre,
+code {
+    background: @code-bg;
+    border-radius: 4px;
+    color: @code-text;
+    font-family: 'Input Mono', Consolas, Monaco, 'Courier New', monospace;
+}
+
+code {
+    display: inline-block;
+    padding: unit( 3px / @size-code, em )
+             unit( 6px / @size-code, em )
+             0;
```

Comment:
Having 0 bottom padding doesn't feel right. What happens if one of the letters has a descender? Does it get uncomfortably close to the bottom edge?